app/app.go

```
@@ -1,42 +1,58 @@
+            Url:        a.MqttUrl,
+            CaCert:     a.MqttCaCert,
+            ClientCert: a.MqttClientCert,
+            ClientKey:  a.MqttClientKey,
+            Ctx:        a.Ctx,
+    }
+
+    if a.MqttEnableTlsKlf && mqttConf.Keylogfile != "" {
+        mqttConf.Keylogfile = a.MqttTlsKlfPath + "_mqtt"
+    }
+
+    mqtt.Create(mqttConf)
```

golangci-lint (1.62.2)

Error return value of `mqtt.Create` is not checked

(errcheck)